### ISSUE TITLE
Short grass showing under snow even though it's disabled in config

### ISSUE THREAD

**Issue by E-daw** [octarine-noise/BetterFoliage#350]
Every time I disable short grass under snow specifically in the config, the grass still shows up underneath the snow. I've tried deleting the config file and having it reload it'self and nothing seems to work
![2021-06-02_20 45 25](https://user-images.githubusercontent.com/34014927/120573622-7d2c9b80-c3e3-11eb-976f-3a137f856892.png)

**Comment by Betestav** [comment]
yeah, can't help myself that much about this, the mod has terrible config problems in general anyways. The leaves configs are even worse
